package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "designsystem",
-  "version": "7.3.0",
+  "version": "7.4.0",
```

Comment:
Core is experimental, and hence in v0.x.y - everything is a patch because the project is in flux and does not yet adhere to SemVer. This is standard practice.

Reply:
The designsystem is minor because the new release contains both features and fixes.